ScoutSuite/__main__.py

```diff
@@ -165,6 +162,7 @@ def main(args=None):
 
 
 def generate_report_name(provider_code, args):
+    report_file_name = {}
```

Comment:
Is there a reason to assign a dictionary to `report_file_name`? Two lines lower, a string is assigned to it. 😕